aii-core/src/main/perl/Shellfe.pm

```diff
@@ -859,6 +895,27 @@ sub check_protected {
     return %hash;
 }
 
+sub change_dhcp 
+{
+    my ($self, $method, %nodes) = @_;
+    $self->debug(5, "logfile:", $self->option('logfile'), " dhcpcfg:", $self->option('dhcpcfg'));
+    my $dhcpmgr = AII::DHCP->new('script',
+        "--logfile=".$self->option('logfile'), 
+        "--cfgfile=".$self->option('dhcpcfg'), 
```

Comment:
@jouvin @jrha this is the important bit. you can also see in the diff that the other options are not part of any removed lines.
maybe the name should be `--aiidhcpcfg` to make it more clear